CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### Changed
 
+- `operator-sdk test local --up-local` test can be run with `--namespace=""` to set environment variable `WATCH_NAMESPACE=""`. (allowed only with `--no-setup` flag)
+
```

Comment:
It should be in the Add. Also, I think it should be something as 🥇 

- Add support to the `operator-sdk test local --up-local` runs with the flag `--namespace=""`. [PR X](URL)

Also, see  its standard. It is missing the PR.

Reply:
Thank you for this :+1: 
done